.binder/postBuild

```diff
@@ -9,7 +9,6 @@
 # - appmode: jupyter extension for executing the notebook
 # - seaborn: visualisation pacakge required for some graphs
 pip install matplotlib pylatexenc pillow appmode seaborn
-pip install .
```

Comment:
I'm guessing this is unrelated, but just fixing an unnecessary config line?  Good to get rid of it, if so.

Reply:
Yeah, I found out that binder was  already installing terra when https://github.com/Qiskit/qiskit-terra/pull/7732/commits/44913fb65eec43176b70eddedbd2d80c993e710c didn't work because rust wasn't installed when binder ran `pip install .` as part of the image build. Figured it's best to remove it here while I was updating everything

Reply:
Yeah, having `pip install .` in a file called "postBuild" seems a bit off.

Reply:
Actually I was mistaken this is needed, the line I saw was from this. I'm not sure why https://github.com/Qiskit/qiskit-terra/commit/44913fb65eec43176b70eddedbd2d80c993e710c didn't work then. But using the conda package does :shrug: